infra/stacks/api.ts

```diff
@@ -2,32 +2,30 @@
 /// <reference path="../../.sst/platform/config.d.ts" />
 
 import heboDatabase from "./db";
+import heboPublicRegistryInfo from "./registry";
+import heboSecurityGroup from "./security-group";
 import heboVpc from "./vpc";
 
 const stackProjectId = new sst.Secret("StackProjectId");
 const stackSecretServerKey = new sst.Secret("StackSecretServerKey");
 
-// Create a security group for the VPC Connector
-const vpcConnectorSecurityGroup = new aws.ec2.SecurityGroup(
-  "VpcConnectorSecurityGroup",
-  {
-    vpcId: heboVpc.id,
-    description: "Security group for App Runner VPC Connector",
-    egress: [
-      {
-        protocol: "-1",
-        fromPort: 0,
-        toPort: 0,
-        cidrBlocks: ["0.0.0.0/0"],
-      },
-    ],
+const heboApiPublicRepo = new aws.ecrpublic.Repository("hebo-api", {
```

Comment:
Is this really public or just the name? Would prefer private, as we don't expose our deployment internals to the world.